File: 은지/LV1/예산.js
function solution(d, budget) {
  var answer = 0;
  const d_sort = d.sort((a, b) => a-b)
  for (let x of d_sort) {
    if (budget >= x) {
      budget -= x
      answer += 1 
    }
  }
  return answer;
}

File: 은지/LV1/예산.py
def solution(d, budget):
    answer = 0
    # 오름차순 정렬
    d.sort()
    for x in d:
        # 예산이 부족한 경우 가지치기
        if budget < x:
            break
        # 예산 사용 가능
        else:
            budget -= x
            answer += 1
    return answer